Tests/BaseObjectTests.swift

```diff
@@ -1,13 +1,16 @@
 import Quick
 import Nimble
-import Clappr
+@testable import Clappr
 
+fileprivate var eventDispatcherPointer: UInt8 = 0
 class BaseObjectTests: QuickSpec {
 
+    class FakeBaseObject: BaseObject { }
+
     override func spec() {
         describe("BaseObject") {
 
-            var baseObject: BaseObject!
+            var fake: FakeBaseObject!
```

Comment:
What we commonly have is `mock` or `stubs`. You can read about their difference here: https://martinfowler.com/articles/mocksArentStubs.html

But in this case, `fake` appears to be just an object that inherits from `BaseObject`. We change this name to `baseObject` or something like that for better comprehension.

Reply:
Hey @leandroalonso, as we talked before, I changed the name of the class to `ConcreteBaseObject` and the name of the var to `baseObject`.